tutorials/09-similarity-sesarch-for-motif-mining.ipynb

```diff
@@ -0,0 +1,724 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    """
</pre><p>Convert these cells to markdown for readibility</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/eva/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>